definitions/subworkflows/sequence_to_bqsr_nonhuman.cwl

```diff
@@ -63,6 +63,9 @@ steps:
         run: ../tools/mark_duplicates_and_sort.cwl
         in:
             bam: name_sort/name_sorted_bam
+            output_name:
+		source: final_name
+                valueFrom: $(self)
```

Comment:
No need to overly complicate this!
```suggestion
            output_name: final_name
```

Reply:
ugh, this is what happens when trying to trace workflow bugs for hours haha